task_base.py

```diff
@@ -86,13 +96,17 @@ class EETask(GeoTask):
     EEDATATYPES = [IMAGECOLLECTION, FEATURECOLLECTION, IMAGE]
 
     @staticmethod
-    def _create_ee_path(asset_path):
+    def _create_ee_path(asset_path, ic=False):
```

Comment:
ic should be spelled out as an arg variable, so it's clear what it is.  also, should a static method be private